src/components/_proxied-dot-io/boundary/footer/style.css

```diff
@@ -10,7 +10,7 @@
 	}
 
 	& a {
-		color: black;
+		color: var(--token-color-palette-neutral-700);
```

Comment:
We don't have a clear value for #000 in the tokens, so I chose the 'darkest' neutral color for now.